frontend/components/TabText/TabText.tsx

```diff
@@ -6,7 +6,9 @@ interface ITabTextProps {
   children: React.ReactNode;
   count?: number;
   /** Changes count badge from default purple to red */
-  isErrorCount?: boolean;
+  redCount?: boolean;
```

Comment:
Nit, but if we want to refactor this we probably don't want to hard-code colors in there (we'll regret it if the colors change), and we have the opportunity to make it more consistent with other components by just having a single `variant` param that you pass a value to.  Like `alert` for red and `neutral` for grey -- something that attempts to represent the state that the colors are trying to convey. What those states are and what colors they correspond to is something that belongs in a design system.